src/Microsoft.DotNet.Build.Tasks.Feed/src/PushToAzureDevOpsArtifacts.cs

```diff
@@ -42,6 +41,12 @@ public override bool Execute()
             {
                 Log.LogMessage(MessageImportance.High, "Performing push to Azure DevOps artifacts storage.");
 
+                if (!string.IsNullOrWhiteSpace(AssetsTemporaryDirectory))
+                {
+                    Log.LogMessage(MessageImportance.High, $"It's no longer necessary to specify a value for the {AssetsTemporaryDirectory} property. " +
```

Comment:
I don't think it's all that useful to print the value instead of the property name
maybe:
```suggestion
                    Log.LogMessage(MessageImportance.High, $"It's no longer necessary to specify a value for the {nameof(AssetsTemporaryDirectory)} property. " +
```